web/server/vue-cli/src/views/Reports.vue

```diff
@@ -375,6 +380,21 @@ export default {
         }
       },
       deep: true
+    },
+    formattedReports: {
+      handler() {
+        this.hasTimeStamp = this.formattedReports.map(
+          report => report.timestamp
+        ).filter(ts => ts).length
+          ? true
+          : false;
+
+        this.hasTestCase = this.formattedReports.map(
+          report => report.testcase
+        ).filter(tc => tc).length
+          ? true
+          : false;
```

Comment:
```suggestion
        this.hasTimeStamp =
          this.formattedReports.some(report => report.timestamp);
        this.hasTestCase =
          this.formattedReports.some(report => report.testcase);
```
May I recommend a shorter implementation?

Reply:
Thank you for the suggestion. Using some() method is more stylish solution.